README.md

```diff
@@ -160,7 +159,7 @@ By the way, `HexInput` is also minimalist-friendly — only 400 bytes gzipped.
 While not only typing its own functions and variables, it can also help you type yours. Depending on the component you're using, you can also import the type that is associated with the component. For example, if you're using our HSL color picker component, you can also import the `HSL` type.
 
 ```ts
-import HslColorPicker, { HSL }  from "react-colorful/hsl";
+import { HslColorPicker, HSL } from "react-colorful";
```

Comment:
With named exports here do we want to find a new way to export the types? I was a fan of our old system as it pretty clearly distinguished the two but I think with named exports this will only get confusing. Now a type (`HSL`) is getting imported in the same way you'd import a component (`HslColorPicker`). 

Not sure I've seen another component library export the types, so I really don't know how we should handle this. New name for them that is less ambiguous, say `typeHsl`? Really not a fan of that name, but can't think of anything better right now.

The exported types are super convenient though, and I do think we should still export them.

Reply:
I'm okay with the current state of color type exports.
We could rename them to make everything more consistent (internally and externally) by renaming them into `HsvColor, HslColor, RgbColor`.

It fits well with our `AnyColor` as well:

```
export type AnyColor = string | HslColor | HsvColor | RgbColor;
```

Reply:
Yeah I like that. Fits better and I think it's also a more clear API.

Reply:
Will do that then.

Reply:
Done